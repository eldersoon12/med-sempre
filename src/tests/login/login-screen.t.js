import { renderComponentTest } from "..";
import Login from "../../pages/login";

renderComponentTest("rendered component", <Login />);
