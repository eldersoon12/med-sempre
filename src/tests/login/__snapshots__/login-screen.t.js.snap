// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendered component 1`] = `
<View>
  <Text>
    Login
  </Text>
</View>
`;
